delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/model/CompositeViewProjection.scala

```diff
@@ -117,7 +119,7 @@ object CompositeViewProjection {
       resourceTag: Option[UserTag],
       includeMetadata: Boolean,
       includeDeprecated: Boolean,
-      includeContext: Boolean,
+      includeContext: Boolean = false,
```

Comment:
There should be a better way to handle this

Reply:
I ended up reverting the decoder changes and writing the function to enrich the `CompositeViewEvents`.